File: packages/skillindex/skills/techs/laravel.ts
export const laravelTech = {
  id: 'laravel',
  name: 'Laravel',
  detect: {
    configFiles: ['artisan', 'bootstrap/app.php'],
    configFileContent: { files: ['composer.json'], patterns: ['"laravel/framework"', '"illuminate/'] },
  },
  skills: ['jeffallan/claude-skills/laravel-specialist', 'affaan-m/everything-claude-code/laravel-patterns'],
};

File: src/skills/techs/laravel.rs
use crate::skills::types::{ConfigFileContentBlock, DetectConfig, Technology};

pub const LARAVEL_TECH: Technology = Technology {
    id: "laravel",
    name: "Laravel",
    detect: DetectConfig {
        packages: &[],
        package_patterns: &[],
        config_files: &["artisan", "bootstrap/app.php"],
        file_extensions: &[],
        gems: &[],
        config_file_content: &[ConfigFileContentBlock {
            files: &["composer.json"],
            patterns: &["\"laravel/framework\"", "\"illuminate/"],
            scan_gradle_layout: false,
            scan_dotnet_layout: false,
        }],
    },
    skills: &[
        "jeffallan/claude-skills/laravel-specialist",
        "affaan-m/everything-claude-code/laravel-patterns",
    ],
};
